### DISCUSSION TITLE
Advice on using GitHub Action

### DISCUSSION THREAD

**Discussion by hhandika** [fzyzcjy/flutter_rust_bridge#1490] [Q&A] [answered]
Hi,

I have just finished migrating to FRB 2.0. I had issues with GitHub Action. 

`info • Dangling library doc comment • rust_builder/cargokit/build_tool/bin/build_tool.dart:1:1 • dangling_library_doc_comments
  error • Target of URI doesn't exist: 'package:build_tool/build_tool.dart' • rust_builder/cargokit/build_tool/bin/build_tool.dart:4:8 • uri_does_not_exist
   info • Dangling library doc comment • rust_builder/cargokit/build_tool/lib/build_tool.dart:1:1 • dangling_library_doc_comments
   info • Dangling library doc comment • rust_builder/cargokit/build_tool/lib/src/android_environment.dart:1:1 • dangling_library_doc_comments
  error • Target of URI doesn't exist: 'package:version/version.dart' `

I wonder if there is any steps that I missed here to get the CI to work.

My CI file

```jobs:
  check:
    name: Test
    runs-on: ${{ matrix.os }}
    strategy:
      matrix:
        os: ["ubuntu-latest", "windows-latest", "macos-latest"]

    steps:
      - name: Check sources
        uses: actions/checkout@v3

      - name: Setup Rust
        uses: dtolnay/rust-toolchain@stable
        with:
          toolchain: stable
      - run: |
          cargo install 'flutter_rust_bridge_codegen@^2.0.0-dev.0'
      - name: Setup Java
        uses: actions/setup-java@v3
        with:
          distribution: "zulu"
          java-version: "17.x"

      - name: Setup Flutter
        uses: subosito/flutter-action@v2
        with:
          channel: "stable"
      - run: |
          flutter pub get
          flutter analyze
      - name: Run test
        uses: subosito/flutter-action@v2
      - run: flutter test
```

Thank you!

**Comment by fzyzcjy** [comment, marked as answer]
That looks like CI is analyzing cargokit code and it fails. The simplest way is to exclude that folder from analyzing. No worries about those errors - cargokit said, after native assets are stablized, it will be a published package.
